Documentation/advanced-configuration.md

```diff
@@ -283,129 +289,90 @@ ceph osd pool set rbd pg_num 512
 With Rook the full swath of
 [Ceph settings](http://docs.ceph.com/docs/kraken/rados/configuration/) are available
 to use on your storage cluster.  When we supply Rook with a ceph.conf file those
-settings will be propagated to all Mon, OSD, etc daemons to use.
+settings will be propagated to all Mon, OSD, MDS, and RGW daemons to use.
 
-In this example we will set the default pool `size` to three, and tell OSD
+In this example we will set the default pool `size` to two, and tell OSD
 daemons not to change the weight of OSDs on startup.
 
-Here's our custom ceph.conf:
+**WARNING**: Modify Ceph settings carefully. You are leaving the sandbox tested by Rook.
+Changing the settings could result in unhealthy daemons or even data loss if used incorrectly.
+
+### Standalone Rook
+
+Here's our custom `ceph.conf` for the OSDs:
 ```bash
 [global]
 osd crush update on start = false
 osd pool default size     = 2
 ```
 
-### Standalone Rook
-
-With our ceph.conf file created we can pass the settings to `rookd` with the
+With our ceph.conf file created we pass the settings to `rookd` with the
 `--ceph-config-override` argument, for example:
 ```bash
 rookd --ceph-config-override=ceph.conf --data-devices=sdf
 ```
 
 ### Kubernetes
-
-With rook-operator on Kubernetes we can edit the OSD DaemonSet after it has been
-created.  This will be streamlined once
-[#571](https://github.com/rook/rook/issues/571) is resolved.
-
-#### Using A ConfigMap
-
-First we will create a ConfigMap that will hold our custom ceph.conf, so it can
-be mounted as a file to be consumed in our Rook pods.
-```bash
-# Create a ConfigMap from a text file
-kubectl -n rook create configmap rook-config --from-file ceph.conf
-
-# View the resulting ConfigMap
-kubectl -n rook get configmap rook-config -o yaml
+When the Rook Operator creates a cluster, a placeholder ConfigMap is created that
+will allow you to override Ceph configuration settings. There are four categories
+of settings, one for each daemon. When the daemon pods are started, the 
+settings specified in this ConfigMap will be merged with the default settings
+generated by Rook. 
+
+The default override settings are blank. Cutting out the extraneous properties, 
+we would see the following defaults after creating a cluster:
+```
+$ kubectl -n rook get ConfigMap config-override -o yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: config-override
```

Comment:
I think @bzub suggested a name change, im suggesting to add a label to the configmap :)